src/main/scala/org/allenai/deep_qa/pipeline/BackgroundCorpusSearcherStep.scala

```diff
@@ -97,14 +95,39 @@ class DefaultBackgroundCorpusSearcherStep(
   override def _runStep() {
     // TODO(matt): might want to make this streaming, for large input files
     val lines = fileUtil.readLinesFromFile(sentencesFile)
-    val indexedSentences = lines.map(line => {
+    val indexedSentences: Seq[(Int, Seq[Query])] = lines.map(line => {
       val fields = line.split("\t")
       sentenceFormat match {
-        case "plain sentence" => (fields(0).toInt, Seq(fields(1)))
+        case "plain sentence" => (fields(0).toInt, Seq(new StringQuery(fields(1))))
         case "question and answer" => {
+          val qText = fields(1)
           val answers = fields(2).split("###")
-          val queries = answers.map(fields(1) + " " + _).toSeq
-          (fields(0).toInt, Seq(fields(1)) ++ queries)
+          val stringQueries = answers
+              .map(a => new StringQuery(qText + " " + a))
+              .toSeq
+          (fields(0).toInt, Seq(new StringQuery(qText)) ++ stringQueries)
+        }
+        case "question and answer boosted" => {
+          // Get the boosting values for question and answer
+          val questionBoost = JsonHelper.extractWithDefault[Float](
```

Comment:
You don't need to have the type parameter here, because it's implied by the third argument.  Also, I would use `1.0` instead of `1f`.